src/omp/complex_min.cpp

```diff
@@ -8,66 +8,54 @@
 
 #include "../complex_min.hpp"
 
-template <typename T>
-struct complex_min<T>::data {
-  std::complex<T>* C;
-};
+template <typename T> struct complex_min<T>::data { std::complex<T> *C; };
 
 template <typename T>
 complex_min<T>::complex_min(long N_) : N(N_), pdata{std::make_unique<data>()} {
   int nthreads = 0;
 
-  #pragma omp parallel
+#pragma omp parallel
```

Comment:
One trick I've played here in the past is to put the indentation between the `#` and the `pragma`:
```c++
# pragma omp parallel
  {
#   pragma omp single
    nthreads = omp_get_num_threads();
  }
```

That might satisfy clang-format, since the `#` gets to stay in column 0.